wiki/start/a-simple-game.md

```diff
@@ -5,605 +5,1091 @@ redirect_from:
 
-Once imported into your IDE, you should have 5 projects or modules: the main one `drop`, and the sub projects `android` (or `drop-android` under Eclipse), `core` / `drop-core`, `lwjgl3` / `drop-lwjgl3`, and `html` / `drop-html`.
+Comments will be used throughout the code examples to explain the facets of this tutorial. You do not have to copy these comments when writing your code:
 
-To launch or debug the game, see the page [Importing & Running a Project](/wiki/start/import-and-running).
+```java
+//This is a comment. It will be ignored by the compiler.
+```
 
-If we just run the project, we will get an error: `Couldn't load file: badlogic.jpg`. Your Run Configuration has to be properly configured first: Select as working directory `PATH_TO_YOUR_PROJECT/​drop/assets`. When we run the game now, we will get the default 'game' generated by the setup app: a Badlogic Games image on a red background. Not too exciting, but that’s about to change.
+However, it is good practice to make your own comments to explain confusing code or to label parts of your design.
 
-## The Game
-The game idea is very simple:
+```java
+//Restrict bucket movement to the width of the viewport.
+```
 
-  * Catch raindrops with a bucket.
-  * The bucket is located in the lower part of the screen.
-  * Raindrops spawn randomly at the top of the screen every second and accelerate downwards.
-  * Player can drag the bucket horizontally via the mouse/touch or move it via the left and right cursor keys.
-  * The game has no end - think of it as a zen-like experience :)
+Import statements are an important part of Java programming. Thankfully, they are automatically added by modern IDE's as you type your code. They are omitted in the examples, but assume they are necessary in your code. They appear at the top of the file under the package statement.
 
-## The Assets
-We need a few images and sound effects to make the game look somewhat pretty. For the graphics we need to define a target resolution of 800x480 pixels (landscape mode on Android). If the device the game is run on does not have that resolution, we simply scale everything to fit on the screen.
+```java
+import com.badlogic.gdx.ApplicationListener;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input.Keys;
+import com.badlogic.gdx.InputProcessor;
+import com.badlogic.gdx.audio.Music;
+import com.badlogic.gdx.audio.Sound;
+```
 
-**Note:** for high profile games you might want to consider using different assets for different screen densities. This is a big topic on its own and won't be covered here.
-{: .notice--primary}
+There are times where something named in libGDX is named the same as something found in another package. As you type `Rectangle` in Idea, for example, it will prompt you to press alt+enter to autocomplete the import. Always opt for the libGDX named class in the subsequent menu.
```

Comment:
You probably should mention that "com.badlogic.gdx" is how you know it's a libGDX class. At this stage most users won't know what AWT even is.

Reply:
Included.